### PR TITLE
Add Places and Desires

### PR DESCRIPTION
**Pull request by MinaSamir1**

### CODE REVIEW

File: src/screens/Desires/AddEditDesire/DesiresController/DesireNameInput.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import React, { useState, useEffect } from 'react';
+import Input from '../../../components/Input/Input';
```
Comment (ahmed-gomaa2) [inline src/screens/Desires/AddEditDesire/DesiresController/DesireNameInput.js:2]: you should use the InputAdmin version of the input here.

File: src/screens/Desires/DesiresPage.js [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+      deleting: isDeletingDesire
+    })
+  }
+
+  ///** The Search View Props */
+  const searchDesiresData = {
+    items: searchDesires,
+    page: 0,
+    searchKey: searchDesireTerm,
+    displayName: 'name',
```
Comment (ahmed-gomaa2) [inline src/screens/Desires/DesiresPage.js:91]: this should be adjusted like the new method created in the Item.js component.

File: src/screens/Places/AddEditPlace/PlaceController/PlacesInput.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
+import React, { useState, useEffect } from 'react';
+import Input from '../../../../components/Input/Input';
```
Comment (ahmed-gomaa2) [inline src/screens/Places/AddEditPlace/PlaceController/PlacesInput.js:2]: you should use the InputAdmin input component.

File: src/screens/Places/PlacesPage.js [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+        setIsDeletingPlace(false);
+    };
+
+
+    ///** The View Props *///
+    const placesData = {
+        items: placeItems,
+        page: page,
+        searchKey: searchPlaceTerm,
+        displayName: 'name',
```
Comment (ahmed-gomaa2) [inline src/screens/Places/PlacesPage.js:75]: this should be adjusted to work with the new display name like the Item.js component.

File: src/screens/Desires/AddEditDesire/DesiresController/DesireNameInput.js
Comment (ahmed-gomaa2) [inline src/screens/Desires/AddEditDesire/DesiresController/DesireNameInput.js:1]: resolve the commented line.

File: src/screens/Desires/DesiresPage.js
Comment (ahmed-gomaa2) [inline src/screens/Desires/DesiresPage.js:1]: please resolve the commented lines.

**Review by ahmed-gomaa2 — COMMENTED** [review]
Please, solve the commented lines!

File: src/screens/Places/PlacesPage.js
[the remark this answers is not in the document]
Comment (MinaSamir1) [reply src/screens/Places/PlacesPage.js:1]: @ahmed-gomaa2 
Thank you for your Review, I have Handled it with some others.

**Pull request merged**